src/main/resources/assets/design-system/form.scss

```diff
@@ -30,3 +30,16 @@
 .sci-token-single-select.token-autocomplete-container.token-autocomplete-singleselect .token-autocomplete-input:after {
   top: 0.5rem;
 }
+
+// DISABLED STATE
+// --------------
+// Disabled and read-only inputs
+input[disabled],
+select[disabled],
+textarea[disabled],
+input[readonly],
+select[readonly],
+textarea[readonly] {
```

Comment:
**Required:** Those are to generic and would also apply to non-scireum fields in websites where our code is loaded. please scope them with a sci class. Like `input.sci-text-input[disabled]` eg.